### PR TITLE
fix(pipeline): reset time ticker immediately before first dbgc

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
reset time ticker immediately before first dbgc

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJhc3NpZ25lZSI6WyIxMDAxMjA1Il19&id=313781&iterationID=-1&pId=0&type=TASK)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | reset time ticker immediately before first dbgc             |
| 🇨🇳 中文    |    在第一次dbgc前就重置下次的时间          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/pipeline/providers/dbgc/dbgc.go [hunk 1]
```diff
@@ -42,13 +42,16 @@ const (
...
-	ticker := time.NewTicker(1 * time.Millisecond)
+	ticker := time.NewTicker(1 * time.Second)
 	defer ticker.Stop()
 	for {
 		select {
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
+			// reset ticker to default for next gc
+			ticker.Reset(p.Cfg.PipelineDBGCDuration)
```
Comment (sfwn) [inline modules/pipeline/providers/dbgc/dbgc.go:53]: issue occurred if dbgc duration is 1 millisecond.

use rutil.continuework

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/pipeline/providers/dbgc/dbgc.go (hunk 1 above)
Comment (chengjoey) [reply modules/pipeline/providers/dbgc/dbgc.go]: adjusted

**Pull request merged**